README.md

```diff
@@ -29,9 +29,7 @@ base URLs and organizational details must conform to the following:
       - Reviewed quarterly and, as necessary, updated
 
 This test kit is provided as a preview for feedback and is not currently
```

Comment:
Lets just remove this whole paragraph.  "Currently" is misleading because it will not be part of (g)(10) certification directly ever, as I understand it.